modules/app/routes/roles.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express')
+const router = express.Router()
+require('passport')
+require('../../authenticationHelpers')
+const controllers = require('../controllers/role')
+const secure = require('./secure')
+
+router.use(secure)
+router.get('/fetch', controllers.fetchRole)
+router.post('/create', controllers.createRole)
+router.delete('/delete', controllers.deleteRole)
+router.put('/update', controllers.updateRole)
+
+module.exports = router
```

Comment:
To have these routes exposed, we need to include on `routes/index.js`

Reply:
Hey @alexanmtz I think I forgot to mention it in the pr 😅 but it is implemented , please check the pr

Reply:
Yeah, sure, I found it 🙏